src/components/inspectors/TimerExpression.vue

```diff
@@ -0,0 +1,381 @@
+<template>
+  <div class="form-group">
+    <label>Start Date</label>
+    <div>
+      <datepicker v-model="startDate"
+                  calendar-class="calendar" format="yyyy-MM-dd"
+                  input-class="form-control start-date" 
+                  class="start-date-div"
+                  @selected="updateStartDate"></datepicker>
```

Comment:
`<datepicker>` can be a self-closing tag

Reply:
Fixed